### PR TITLE
Update names for Polish neighbourhood

### PR DESCRIPTION
**Pull request by stepps00**
This PR updates names for a neighbourhood in Poland. All existing names are stored as variants, and new preferred names have been added. No PIP work needed.

### CODE REVIEW

File: data/149/500/582/3/1495005823.geojson [hunk 1]
```diff
@@ -22,21 +22,54 @@
...
+        "\u0413\u043e\u0441\u0446\u0438\u0446\u0438\u043d\u043e"
+    ],
+    "name:csb_x_preferred":[
+        "G\u00f2sc\u00ebc\u00ebno"
+    ],
     "name:eng_x_preferred":[
         "Go\u015bcicino"
     ],
+    "name:eng_x_variant":[
+        "Go\u015bcicino"
```
Comment (nvkelso) [inline data/149/500/582/3/1495005823.geojson:41]: This is the same as the Eng preferred, so the Eng variant should be dropped.

File: data/149/500/582/3/1495005823.geojson [hunk 2]
```diff
@@ -22,21 +22,54 @@
...
         "Go\u015bci\u0119cino"
     ],
     "name:pol_x_preferred":[
+        "Go\u015bcicino"
+    ],
+    "name:pol_x_variant":[
         "Go\u015bci\u0119cino"
     ],
-    "name:rus_x_preferred":[
+    "name:rus_x_variant":[
```
Comment (nvkelso) [inline data/149/500/582/3/1495005823.geojson:58]: If it's the only one, then Rus preferred

**Review by nvkelso — CHANGES_REQUESTED** [review]
Nits

**Comment by stepps00** [comment]
Good catches.. I've updated the record to reflect these comments. The record should now be ready to merge.

File: data/149/500/582/3/1495005823.geojson [hunk 3]
```diff
@@ -99,8 +129,8 @@
         }
     ],
     "wof:id":1495005823,
-    "wof:lastmodified":1570060895,
-    "wof:name":"Go\u015bci\u0119cino",
+    "wof:lastmodified":1652374652,
+    "wof:name":"Go\u015bcicino",
```
Comment (nvkelso) [inline data/149/500/582/3/1495005823.geojson:133]: NIt: Is there an ASCII7 version?

**Review by nvkelso — APPROVED** [review]

File: data/149/500/582/3/1495005823.geojson (hunk 3 above)
Comment (stepps00) [reply data/149/500/582/3/1495005823.geojson]: Updated in a more recent commit..

**Review by nvkelso — APPROVED** [review]

**Pull request merged**